api/invite.go

```diff
@@ -532,3 +583,112 @@ func (a *Api) SendInvite(res http.ResponseWriter, req *http.Request, vars map[st
 	}
 	return
 }
+
+func (a *Api) SendTeamInvite(res http.ResponseWriter, req *http.Request, vars map[string]string) {
+	// By default, the invitee language will be "en" for Englih (as we don't know which language suits him)
+	// In case the invitee is a known user, the language will be overriden in a later step
+	var inviteeLanguage = "en"
+	if token := a.token(res, req); token != nil {
```

Comment:
```suggestion
	token := a.token(res, req)
	if token == nil {
	    return
	}
```
Just to avoid the big if block ;)

Reply:
sounds like a good point 
will make the update where it can be done